client/src/components/ElementEditor/AddElementPopover.js

```diff
@@ -3,6 +3,7 @@ import { Button, Input, InputGroup, InputGroupAddon, Popover } from 'reactstrap'
 import classNames from 'classnames';
 import { elementTypeType } from 'types/elementTypeType';
 import i18n from 'i18n';
+import addElementMutation from 'state/editor/addElementMutation';
```

Comment:
We normally apply these HOC's with Injector. This allows people to sub out this query for their own if need be.

Reply:
To be fair, this pattern is how we've done it before, but we _should_ be aiming to make them injectable where possible. Related: https://github.com/silverstripe/silverstripe-admin/issues/611

Reply:
Was implemented as directed, however have updated it anyway as I also think it's a good idea.